### DISCUSSION TITLE
Problem z połączeniem ACI do lokalnego Rabbita

### DISCUSSION THREAD

**Discussion by szymko93** [AzureCommunityPL/group#366] [Pomoc]
Cześć, mam problem z nawiazaniem polaczenia ACI do rabbita w lokalnej sieci. W Azure mam dwie subskrypcje. W jednej postawiony mam Container Instances a w drugiej VPN do on prema. Pomiędzy VNETami zrobiłem peering (status connected). I teraz zaczyna się problem bo tunel wraz z portami mam również udrożniony do lokalnych zasobów, czyli teoretycznie przejście z ACI do rabbita jest. Natomiast kiedy uruchamia kontener dostaje komunikat "connections failed" . Dodam, że VNETy są w różnych sieciach 172 i 192. Ktoś ma jakiś pomysłjak się dostać do rmq? NAT Gateway?

**Comment by piotrgwiazda** [comment]
Tu są jakieś nie wspierane scenariusze w tym global peering
https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts


pon., 18 paź 2021, 18:38 użytkownik Szymon ***@***.***>
napisał:

**Reply by KamilWiecek** [reply]
"connections failed" to trochę za mało, żeby coś Ci doradzić. 

Jeżeli wykluczysz niewspierane scenariusz jak napisał @piotrgwiazda, to podaj więcej szczegółów: jak wygląda routing, ustawienia peeringu i konfiguracja VPN Gateway. Czy cokolwiek znajdującego się w sieci spoke widzi się z czymkolwiek po stronie on prem?

Przyczyn może być wiele w tym niektóre całkiem trywialne jak np. rozwiązanie nazwy DNS, czy firewall na Rabbicie, aż po NSG. Łatwiej będzie Ci przeprowadzić troubleshooting z poziomu ACI, a jeszcze łatwiej z poziomu VM w tym samym VNecie.

**Comment by mgajownik** [comment]
obie sieci azure 172 i 192 masz dodane w konfiguracji routera onprem? i obie subscrypcje maja ta samo directory?

**Reply by szymko93** [reply]
Na routerze jest dodana tylko 192. 
Obie mają to samo dierctory

**Comment by piotrgwiazda** [comment]
W kwestii debugowania nie jestem pewien czy ACI wpięty w sieć działa tak
samo jak VM, która ma interfejs sieciowy wpięty w subnet.

Jeśli peering nie działa dla ACI, to może jakiś load balancer postawić jako
przejściówkę. A może zbudować sobie Private Link Service po stronie Rabbita
i zarejestrować Private Endpoint tak jak się wpina PaaS-y w sieć. PE będzie
miał IP z subnetu, w ktory wpięty jest ACI.

wt., 19 paź 2021, 20:22 użytkownik amorficzny ***@***.***>
napisał:
